package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unly/serverless-plugin-db-backups",
```

Comment:
You forgot to rename the package before publishing, I guess we can change the name and publish the new one, probably should delete the existing one afterwards to avoid confusion. Also need to rename all usages.